less/slider.less

```diff
@@ -143,7 +144,7 @@
             .dir-rtl & {
                 transform: rotateY(180deg);
                 -webkit-transform: rotateY(180deg);
-                -ms-transform: rotateY(180deg);
+                -ms-transform: rotate(180deg);
```

Comment:
This is reintroducing a typo which @guywillis fixed in https://github.com/adaptlearning/adapt-contrib-slider/commit/72a43bb79e9b0336771d2d5f1c0c374b5d713f56.